lib/getJobsCompletedCount.ts

```diff
@@ -14,34 +11,20 @@ export default async (): Promise<number> => {
 }
 
 const updateCache = async (kv: Deno.Kv): Promise<number> => {
-  const jobsCompleted = await getFromInflux()
+  const jobsCompleted = await getFromApi()
   await kv.set(['jobs-completed'], jobsCompleted)
   return jobsCompleted
 }
 
-const OFFSET = 567_564_171
-const OFFSET_DATE = '2024-03-03T17:14:30.236298505Z'
+const OFFSET = 155_153_052_475
+const OFFSET_DATE = '2024-07-09'
 
-export const getFromInflux = async (): Promise<number> => {
-  const res = await fetch("https://eu-central-1-1.aws.cloud2.influxdata.com/api/v2/query", {
-    method: "POST",
-    headers: {
-      "Accept": "application/csv",
-      "Content-Type": "application/vnd.flux",
-      "Authorization": `Token ${Deno.env.get("INFLUX_TOKEN")}`,
-    },
-    body: `
-      from(bucket: "station")
-        |> range(start: ${OFFSET_DATE})
-        |> filter(fn: (r) => r["_measurement"] == "jobs-completed")
-        |> filter(fn: (r) => r["_field"] == "value")
-        |> group()
-        |> sum()
-    `,
-  })
-  const body = await res.text()
-  assert(res.ok, `Bad InfluxDB response: ${res.status} ${res.statusText}`)
-  return OFFSET + Number(
-    body.split('\n')[1].split(',')[5]
+export const getFromApi = async (): Promise<number> => {
+  const res = await fetch(`https://stats.filspark.com/measurements/daily?from=${OFFSET_DATE}`)
```

Comment:
Do you have any concerns about the performance here?

ATM, the request takes 17 seconds to complete, and this time will grow proportionally to the number of days queried.

The situation should improve once we fix https://github.com/space-meridian/roadmap/issues/132.

Reply:
Potential improvement we can implement periodically:
- Query the jobs completed from OFFSET_DATE until YESTERDAY
- Update OFFSET and OFFSET_DATE so that the next query starts from today

Reply:
At minimum, I think we need a way to monitor the duration of these queries. We can keep this super simple, e.g. printing the duration to logs. (Assuming there is a way to view logs of functions deployed to Deno Deploy. Is that true?)

Reply:
> Do you have any concerns about the performance here?

This query is cached by cloudflare, so I don't think this is adding more load than the public dashboard is already creating.

But you're right that the longer ago `OFFSET_DATE` will be, the more expensive this query will be. I was thinking about instead running many queries, one per day. This way the day query can be cached by cloudflare indefinitely. Wdyt about that?

> Potential improvement we can implement periodically

Good idea! It adds complexity and state but should keep the queries small. What do you think about that versus making queries per day?

> At minimum, I think we need a way to monitor the duration of these queries. We can keep this super simple, e.g. printing the duration to logs. (Assuming there is a way to view logs of functions deployed to Deno Deploy. Is that true?)

Yes! We have logs in deno deploy. Added in #05d9c2c

Reply:
> I was thinking about instead running many queries, one per day. This way the day query can be cached by cloudflare indefinitely. Wdyt about that?

I think this is the best option, as it should be (relatively) easy to implement and does not require ongoing maintenance.

:clap:

OTOH, I hope we will finish the database schema changes in 1-3 weeks; we can even prioritise the work on speeding up the total job count as the first thing to do. Once deployed, I expect the query to complete in under a second regardless of the number of days we query (within a reasonable limit, e.g. one year). So if you move the snapshot date to yesterday, we may not need to change the snapshot again until next summer, if at all.

Reply:
Agreed, let's keep it like this, since we know how to improve it easily in the future